oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/parser/MetricsHolder.java

```diff
@@ -26,10 +26,19 @@
 import org.apache.skywalking.oap.server.core.analysis.metrics.Metrics;
 import org.apache.skywalking.oap.server.core.analysis.metrics.annotation.MetricsFunction;
 
+@SuppressWarnings("UnstableApiUsage")
 public class MetricsHolder {
-    private static Map<String, Class<? extends Metrics>> REGISTER = new HashMap<>();
+    private static final Map<String, Class<? extends Metrics>> REGISTER = new HashMap<>();
 
-    public static void init() throws IOException {
+    static {
+        try {
+            init();
+        } catch (IOException e) {
+            throw new RuntimeException("load metrics functions error.", e);
+        }
+    }
```

Comment:
Please make this as lazy initialization in the `#find` method. The static block is not recommended.